src/itemChoice/index.tsx

```diff
@@ -75,7 +75,6 @@ class ItemChoice extends PureComponent<ItemChoiceProps> {
       prefix({
         itemChoice: true,
         'itemChoice--highlighted': highlighted,
-        'itemChoice--withSubLabel': !!subLabel,
```

Comment:
Since we're talking about refactoring ItemChoice, I'd prefer to change the style in the app rather than in the library.
Something like:
`.ItemChoiceList > li:first-child {
    padding-top: 16px;
}`

Reply:
Acutally, we should never target the classes and change the CSS/design inside the library component from the app

Reply:
Ok I misinterpreted what you needed the change for.